### PR TITLE
Adds a sprite for advanced medkit medibots

### LINKED ISSUE #51694: medbots made with /obj/item/storage/firstaid/advanced need a unique overlay sprite made
advanced medibot

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
image

Closes #51694

## Changelog
:cl:
imageadd: Advanced medibots now have a distinct sprite.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
thanks

**Comment by Jared-Fogle** [comment]
when will someone show appveyor where the rabbits are

**Comment by Tlaltecuhtli** [comment]
is there one for nook op?

**Comment by Fikou** [comment]
> is there one for nook op?

yeah but it uses the wrong colors since medkit was updated to slick black but medbot stayed a weird desaturated purple

**Review by Twaticus — APPROVED** [review]

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -300,14 +300,16 @@
...
-	else if(istype(src, /obj/item/storage/firstaid/toxin))
+	else if (istype(src, /obj/item/storage/firstaid/toxin))
 		A.set_skin("tox")
-	else if(istype(src, /obj/item/storage/firstaid/o2))
+	else if (istype(src, /obj/item/storage/firstaid/o2))
 		A.set_skin("o2")
-	else if(istype(src, /obj/item/storage/firstaid/brute))
+	else if (istype(src, /obj/item/storage/firstaid/brute))
 		A.set_skin("brute")
+	else if (istype(src, /obj/item/storage/firstaid/advanced))
```
Comment (Rohesie) [inline code/game/objects/items/storage/firstaid.dm:311]:
```
The following coding styles are not only not enforced at all, but are generally frowned upon to change for little to no reason:
paces after control statements
    if() and if () - nobody cares!
```
Don't add extra history for no reasons

**Pull request merged**